### PR TITLE
Fallback to default config to enable testing Image component

### LINKED ISSUE #18415: Unable to test components that import "next/image"
# Bug report

## Describe the bug

Using jest and react-testing-library, tests that run on components that use the new Image component fail with the following error:

```
    TypeError: Cannot destructure property 'deviceSizes' of 'imageData' as it is undefined.

    > 1 | import Image from "next/image";
        | ^
      2 | 
```

## To Reproduce

As far as my testing has gone just simply running a test against any component that imports Image from "next/image" causes this error. I've setup a simple vanilla nextjs repo that adds jest and react-testing-library, along with a component that uses the Image component here: https://github.com/spencewood/next-image-test. Running the tests on this repo yields the above error:

```
> yarn test
```

## Expected behavior

I would expect the tests to run normally.

## System information

- OS: macOS 10.15
- Version of Next.js: 10.0
- Version of Node.js: 12.18

### PR DESCRIPTION
**Pull request by styfle**
Fixes #18415 by using the default config as fallback.

Users who wish to use their `next.config.js` values will still need the workaround from https://github.com/vercel/next.js/issues/18415#issuecomment-718180659

### CODE REVIEW

File: test/integration/image-optimizer/test/index.test.js [hunk 1]
```diff
@@ -546,6 +546,31 @@ describe('Image Optimizer', () => {
...
+      let stderr = ''
+
+      app = await launchApp(appDir, await findPort(), {
+        onStderr(msg) {
+          stderr += msg || ''
+        },
+      })
+      await waitFor(1000)
+      await killApp(app).catch(() => {})
+      await nextConfig.restore()
```
Comment (Timer) [inline test/integration/image-optimizer/test/index.test.js:568]: This should probably be in a finally

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| buildDuration | 13.2s | 13.2s | ⚠️ +28ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +12.9 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.48 | 2.385 | -0.1 |
| / avg req/sec | 1008.12 | 1048.09 | +39.97 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.312 | 1.296 | -0.02 |
| /error-in-render avg req/sec | 1905.1 | 1929.19 | +24.09 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..d4b6.js gzip | 12.6 kB | 12.6 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-43dd9e5..8439.js gzip | 6.48 kB | 6.48 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.8 kB | 58.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.22 kB | 8.22 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-472f7d5..dule.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.5 kB | 53.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| index.html gzip | 970 B | 970 B | ✓ |
| link.html gzip | 977 B | 977 B | ✓ |
| withRouter.html gzip | 962 B | 962 B | ✓ |
| Overall change | 2.91 kB | 2.91 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| buildDuration | 15s | 15.4s | ⚠️ +443ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +2.92 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..d4b6.js gzip | 12.6 kB | 12.6 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-43dd9e5..8439.js gzip | 6.48 kB | 6.48 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.8 kB | 58.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.22 kB | 8.22 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-472f7d5..dule.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.5 kB | 53.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _error.js | 916 kB | 916 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 916 kB | 916 kB | ✓ |
| link.js | 974 kB | 974 kB | ✓ |
| routerDirect.js | 967 kB | 967 kB | ✓ |
| withRouter.js | 967 kB | 967 kB | ✓ |
| Overall change | 4.75 MB | 4.75 MB | ✓ |

</details>

</details>
Commit: aebb52b350e3c67ee25ccbaddeb61e496a72df85

File: test/integration/image-optimizer/test/index.test.js (hunk 1 above)
Comment (styfle) [reply test/integration/image-optimizer/test/index.test.js:568]: I'm following the pattern from the other tests.

Are you suggesting remove the `.catch()` above and instead wrap the killApp in a try/finally?

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.1s | -216ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +2.92 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.369 | 2.305 | -0.06 |
| / avg req/sec | 1055.3 | 1084.43 | +29.13 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.313 | 1.261 | -0.05 |
| /error-in-render avg req/sec | 1903.99 | 1982.58 | +78.59 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..d4b6.js gzip | 12.6 kB | 12.6 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-43dd9e5..8439.js gzip | 6.48 kB | 6.48 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.8 kB | 58.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.22 kB | 8.22 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-472f7d5..dule.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.5 kB | 53.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| index.html gzip | 970 B | 970 B | ✓ |
| link.html gzip | 977 B | 977 B | ✓ |
| withRouter.html gzip | 962 B | 962 B | ✓ |
| Overall change | 2.91 kB | 2.91 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| buildDuration | 15.3s | 15.3s | -4ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +2.92 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..d4b6.js gzip | 12.6 kB | 12.6 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-43dd9e5..8439.js gzip | 6.48 kB | 6.48 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.8 kB | 58.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.22 kB | 8.22 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-472f7d5..dule.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.5 kB | 53.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _error.js | 916 kB | 916 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 916 kB | 916 kB | ✓ |
| link.js | 974 kB | 974 kB | ✓ |
| routerDirect.js | 967 kB | 967 kB | ✓ |
| withRouter.js | 967 kB | 967 kB | ✓ |
| Overall change | 4.75 MB | 4.75 MB | ✓ |

</details>

</details>
Commit: 327bfff62700e4588ff082f8ea3ddf88dba8ab4f

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| buildDuration | 11.3s | 11.4s | ⚠️ +41ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +2.92 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.126 | 2.148 | ⚠️ +0.02 |
| / avg req/sec | 1175.83 | 1164.12 | ⚠️ -11.71 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.173 | 1.139 | -0.03 |
| /error-in-render avg req/sec | 2130.93 | 2195.02 | +64.09 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-43dd9e5..8439.js gzip | 6.48 kB | 6.48 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-472f7d5..dule.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.5 kB | 53.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| index.html gzip | 972 B | 972 B | ✓ |
| link.html gzip | 977 B | 977 B | ✓ |
| withRouter.html gzip | 964 B | 964 B | ✓ |
| Overall change | 2.91 kB | 2.91 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| buildDuration | 13.1s | 13.1s | ⚠️ +90ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +2.92 kB |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-43dd9e5..8439.js gzip | 6.48 kB | 6.48 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-472f7d5..dule.js gzip | 5.58 kB | 5.58 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.5 kB | 53.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | vercel/next.js 18415-test-default-image | Change |
| - | - | - | - |
| _error.js | 916 kB | 916 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 916 kB | 916 kB | ✓ |
| link.js | 974 kB | 974 kB | ✓ |
| routerDirect.js | 967 kB | 967 kB | ✓ |
| withRouter.js | 967 kB | 967 kB | ✓ |
| Overall change | 4.75 MB | 4.75 MB | ✓ |

</details>

</details>
Commit: 18a974d2a9e8a99ec046b4d918fbca9def92d91e

File: test/integration/image-optimizer/test/index.test.js (hunk 1 above)
Comment (Timer) [reply test/integration/image-optimizer/test/index.test.js:568]: We automatically retry tests in CI so it's important that these tests clean themselves up:
```tsx
it("should error when loader contains invalid value", async () => {
  await nextConfig.replace(
    "{ /* replaceme */ }",
    JSON.stringify({
      images: {
        loader: "notreal",
      },
    })
  );
  let stderr = "";
  try {
    app = await launchApp(appDir, await findPort(), {
      onStderr(msg) {
        stderr += msg || "";
      },
    });
    await waitFor(1000);
    await killApp(app).catch(() => {});

    expect(stderr).toContain(
      "Specified images.loader should be one of (default, imgix, cloudinary, akamai), received invalid value (notreal)"
    );
  } finally {
    await nextConfig.restore();
  }
});
```

**Review by Timer — APPROVED** [review]

**Pull request merged**